src/app/components/mnoe-dashboards-list/mnoe-dashboards-list.coffee

```diff
@@ -66,5 +66,12 @@
     vm.toggleDashboardKpis = (dashboard) ->
       dashboard.kpiOpened = !dashboard.kpiOpened
 
+    vm.getKpiName = (kpi) ->
+      kpiName = kpi.endpoint.split('/').pop()
+      localeName = "mnoe_admin_panel.dashboard.dashboard_list.widget.list.kpis.#{kpiName}"
+      # $translate.instant returns key if key is not found.
```

Comment:
This is a tad hacky, but $translate doesn't provide ability to use fallback locale for `$translate.instant`, and I didn't want to use `$translate`, as with the promises, it would be a headache. https://angular-translate.github.io/docs/#/api/pascalprecht.translate.directive:translate

Reply:
Happy with this solution